ee/api/performance_events.py

```diff
@@ -157,3 +207,15 @@ def list(self, request: request.Request, *args, **kwargs) -> Response:
         serializer.is_valid(raise_exception=True)
 
         return Response({"results": serializer.data})
+
+    @action(methods=["GET"], detail=False)
+    def recent_pageviews(self, request: request.Request, *args, **kwargs) -> Response:
+        params_serializer = RecentPageViewPerformanceEventsQuerySerializer(data=request.GET)
+        params_serializer.is_valid(raise_exception=True)
+        params = params_serializer.validated_data
+
+        results = RecentPageViewPerformanceEvents.query(self.team_id, number_of_days=params.get("number_of_days"))
```

Comment:
Totally up for this given it is all just being used by us atm but before we go live with any of this we should probably try and conform to the typical API `date_from`

Reply:
fair! 👍